src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/WebUI.cs

```diff
@@ -90,5 +113,14 @@ private void CallbackMethod(AuthorizationResult result)
         {
             SetAuthorizationResult(result);
         }
+
+        public void Dispose()
+        {
+            if (this.taskId != UIApplication.BackgroundTaskInvalid)
+                UIApplication.SharedApplication.EndBackgroundTask(this.taskId);
```

Comment:
why this is necessary?

Reply:
I just wanted to be certain that the events are unregistered if the webUI is disposed before the app is returned to the foreground somehow. probably not needed.